src/content/en/fundamentals/accessibility/index.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: shared/narrow-subdirectories-list
+title: "Accessibility"
+description: "Improving accessibility for web pages"
+published_on: 2016-06-27
+updated_on: 2016-06-27
+translation_priority: 0
+order: 6
+---
+
+This document set is a text-based version of part of the content covered in the Udacity course on Accessibility. Rather than a direct transcription of the video course, it is meant to be a more concise treatment of accessibility principles and practices, using the course's original content as a base.
```

Comment:
[the Udacity course on Accessibility](https://www.udacity.com/course/web-accessibility--ud891)